app/views/api/v2/hosts/reported_data.json.rabl

```diff
@@ -0,0 +1,7 @@
+glue(@facet) do
+  attributes :uptime_seconds
+end
+
+child(@facet => :reported_data) do
+  attributes :uptime_seconds, :cores, :sockets, :disks_total
```

Comment:
Can we use the boot time? We may need to keep `uptime_seconds` in for compatibility and don't have a good way of deprecating it, but the boot time is a stable value while uptime in seconds will change on every request. It may even be a lie: the host could be down and Foreman would still continue to calculate a new uptime. IMHO users should be using the boot time.

Reply:
ofc, that was what I meant to add :facepalm: Thanks I've kept the uptime_seconds in the root namespace for backward compatibility and the boot_time is under the `reported_data`